packages/api-server/src/ws/methods.ts

```diff
@@ -73,16 +77,29 @@ export function wrapper(ws: any, req: any) {
     const callback = (err: any, result: any) => {
       return { err, result };
     };
+
+    // check batch limit
+    const errs = wsBatchLimit(objs);
+    if (errs != null) {
+      return cb(
```

Comment:
Does the `args` handled by Node.js net.js function normalizeArgs?

code ref: net.js?L252:10
https://sourcegraph.com/github.com/nodejs/node/-/blob/lib/net.js?L252:10&popover=pinned